diff --git a/docker/build_scripts/build.sh b/docker/build_scripts/build.sh
index a6eb23d..5b223e9 100755
--- a/docker/build_scripts/build.sh
+++ b/docker/build_scripts/build.sh
@@ -165,7 +165,7 @@ yum -y clean all > /dev/null 2>&1
 yum list installed
 
 # we don't need libpython*.a, and they're many megabytes
-find /opt/_internal -name '*.a' -print0 | xargs -0 rm -f
+# find /opt/_internal -name '*.a' -print0 | xargs -0 rm -f
 
 # Strip what we can -- and ignore errors, because this just attempts to strip
 # *everything*, including non-ELF files:
